src/user/announcements.rst

```diff
@@ -9,6 +9,23 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 ----
 
 
+2022-08-11: Moving to Visual Studio toolchain vc142
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Microsoft has deprecated the Visual Studio (VS) 2017 compiler and removed it
+from all the CI they control (notably Azure Pipelines & Github Actions).
+This means that the default toolchain (== C++ compiler, linker, standard
+libraries, and related utilities) of that VS version - vc141 - is getting less
+and less use in upstream libraries (because their public CI doesn't use it
+anymore without a lot of fiddling), and therefore support for it is bitrotting
```

Comment:
```suggestion
and less use in upstream libraries (because public hosted CI doesn't use it
anymore by default), and therefore support for it is bitrotting
```